platform.txt

```diff
@@ -129,3 +129,26 @@ tools.openocd.erase.pattern=
 tools.openocd.bootloader.params.verbose=-d2
 tools.openocd.bootloader.params.quiet=-d0
 tools.openocd.bootloader.pattern="{path}/{cmd}" {bootloader.verbose} -f interface/{program.protocol}.cfg -c "{program.setup_command}" -f target/{upload.target}.cfg -c "init; halt; nrf51 mass_erase; program {{{runtime.platform.path}/cores/nRF5/SDK/components/softdevice/{softdevice}/hex/{softdevice}_{upload.target}_{softdeviceversion}_softdevice.hex}} verify reset; shutdown;"
+
+# blackmagic probe upload
+tools.bmp_upload.cmd=arm-none-eabi-gdb
+tools.bmp_upload.path={runtime.tools.gcc-arm-none-eabi-5_2-2015q4.path}/bin/
+
+tools.bmp_upload.upload.speed=230400
```

Comment:
Do different boards need different values?

Reply:
Not that ive noticed. Ive tried two different nrf51822 chips over 3 boards and have never had an issue.

@rogerclarkmelbourne - have you had any problems with this upload speed with your nRF51 or nRF52 boards?